boards/arm/custom_nrf52840dk/Kconfig

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
> easier for users to update than inconsistent spot changes.

You and I continue to disagree on this. Your perspective seems to be that this example should be some sort of template generator the user can run a script on to customize, while I continue to maintain that [short names that are unlikely to conflict](https://github.com/zephyrproject-rtos/zephyr/issues/28438#issuecomment-694489513) are better for readability.

Reply:
I don't see the naming problem here, it just demonstrates the creation of a custom board. It can have any name, just chose this one, but I'm open to other suggestions.

Reply:
Regarding the board name.
In the official documentation we use the board name [plank](https://docs.zephyrproject.org/latest/guides/porting/board_porting.html#create-your-board-directory), so I would like to suggest to use either `plank` or alternatively `custom_plank` in this example.

That will make it easier for readers to relate the docs with this board example, and also search for `plank`:
https://docs.zephyrproject.org/latest/search.html?q=plank&check_keywords=yes&area=default